src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/TilsynBarnBeregnYtelseSteg.kt

```diff
@@ -31,7 +31,7 @@ class TilsynBarnBeregnYtelseSteg(
 ) {
 
     override fun lagreVedtak(saksbehandling: Saksbehandling, vedtak: InnvilgelseTilsynBarnDto) {
-        val beregningsresultat = tilsynBarnBeregningService.beregn(vedtak.stønadsperioder, vedtak.utgifter)
+        val beregningsresultat = tilsynBarnBeregningService.lagBeregningsgrunnlagOgBeregn(behandlingId = saksbehandling.id, vedtak.utgifter)
```

Comment:
Tenker `beregn` er tilstrekkelig?

Reply:
Hmm jaa holder egentlig, men er ikke så fan av at metoder heter det samme og de skiller de seg fra hverandre ved at den ene lager beregningsgrunnlag i tillegg. Tenkte det kanskje kom frem litt tydligere at funksjonen også henter inn info den trenger for å beregne